src/query/storages/parquet/src/parquet_rs/parquet_reader.rs

```diff
@@ -123,6 +126,8 @@ impl ParquetRSReader {
             options,
         )?;
 
+        let batch_size = ctx.get_settings().get_max_block_size()? as usize;
```

Comment:
it seems `max_block_size` is currently used mainly for join, i.e.  temporary blocks.
but when used in CopyIntoTable, it is better to read with the BlockThreadsHolds of the dest fuse table, to avoid split/compact again before write.

Reply:
If there isn't a dest table, how can I determine the batch size?

Reply:
we can use max_block_size then.
we can even impl choosing batch size for copy in next pr.